espn_api/basketball/league.py

```diff
@@ -104,6 +105,21 @@ def recent_activity(self, size: int = 25, msg_type: str = None, offset: int = 0,
 
         return activity
 
+    def transactions(self, scoring_period: int = None) -> List[Transaction]:
+        '''Returns a list of recent transactions'''
+        params = {
+            'view': 'mTransactions2',
+            'scoringPeriodId': scoring_period,
+        }
+
+        filters = {"transactions":{"filterType":{"value":["FREEAGENT","WAIVER","WAIVER_ERROR"]}}}
```

Comment:
I think having these filter types as default is good and then allow it to be passed in to the function as well in case you only want to look for one or different ones.